modules/apiserversource-kn.adoc

```diff
@@ -83,16 +83,18 @@ Conditions:
 
 .Verification
 
-You can verify that the Kubernetes events were sent to Knative by looking at the message dumper function logs.
+To verify that the Kubernetes events were sent to Knative, you can look at the event-display logs or use web browser to see the events.
 
-. Get the pods:
+. To view the events in a web browser, open the link returned by the following command:
 +
 [source,terminal]
 ----
-$ oc get pods
+$ kn service describe event-display -o url
```

Comment:
We didn't mention "event-display" service anywhere before. We suggested exactly this: `kn service create <service_name> --image quay.io/openshift-knative/showcase` . So, maybe we could suggest a concrete service name at the beginning and at this point use the concrete name as well.

Reply:
I.e. let's use this at the beginning: `kn service create event-display --image quay.io/openshift-knative/showcase` and then we can reference it as "event-display" later.